src/mlio-py/mlio/core/data_reader.cxx

```diff
@@ -223,6 +223,12 @@ make_recordio_protobuf_reader(data_reader_params rdr_prm)
     return make_intrusive<recordio_protobuf_reader>(std::move(rdr_prm));
 }
 
+intrusive_ptr<text_line_reader>
+make_text_line_reader(data_reader_params rdr_prm)
```

Comment:
You can rename `rdr_prm` to `prm` as this is the only parameter of the constructor.